screwdriver.yaml

```diff
@@ -0,0 +1,35 @@
+shared:
+    image: golang:1.15.5
```

Comment:
We should use latest go version for new component if there is no reason. (my previous comment was resulted just refactor sd.yaml 🙇 )
https://github.com/screwdriver-cd/sd-local/pull/75